### PR TITLE
Remove the unused implicit attribute `_cc_toolchains`.

### PR DESCRIPTION
**Pull request by katre**
CC toolchains are gathered from the line:
```
toolchains = use_cc_toolchains(),
```

Keep toolchain usage of _cc_toolchain with a TODO to remove it.

See #3115 for details.

### CODE REVIEW

File: rust/toolchain.bzl [hunk 1]
```diff
@@ -874,6 +874,7 @@ rust_toolchain = rule(
                 "For more details see: https://docs.bazel.build/versions/master/skylark/rules.html#configurations"
             ),
         ),
+        # TODO: #3115 - Remove this attribute.
```
Comment (illicitonion) [inline rust/toolchain.bzl:877]: Why aren't we removing this one too? When should we / what's blocking doing so?

**Review by illicitonion — APPROVED** [review]
Thanks!

File: rust/toolchain.bzl (hunk 1 above)
Comment (illicitonion) [reply rust/toolchain.bzl:877]: AFAICT `toolchain.crosstool_files` is unused in our codebase, so we should be able to just drop it.

**Comment by katre** [comment]
Can this be merged as-is, and I'll send another PR to try removing `toolchain.crosstool_files`?

**Comment by illicitonion** [comment]
Works for me! Thanks!

**Pull request merged**